python/pysrc/kaskada/_timestream.py

```diff
@@ -1192,6 +1194,69 @@ def run_iter(
 
         raise AssertionError(f"Unhandled kind {kind}")
 
+    def plan(
+        self,
+        kind: Literal["initial_dfg", "final_dfg", "final_plan"] = "final_plan",
+        results: Optional[Union[kd.results.History, kd.results.Snapshot]] = None,
+        format: Literal["dot", "svg", "svg_str"] = "svg",
+        mode: Literal["once", "live"] = "once",
+    ) -> Union[str, "IPython.display.SVG"]:
```

Comment:
We could have special cases for the return based on the `format`, but I don't think we need that strict of types here yet.